src/mesh/mod.rs

```diff
@@ -0,0 +1,234 @@
+    }
+
+    fn quad([left, right]: [Vec3; 2], normal: Vec3, height: f32) -> Self {
+        let offset = BASE_FACING * height;
+        Self {
+            vertices: vec![left, left + offset, right + offset, right],
+            normals: [normal; 4].to_vec(),
+            uvs: vec![Vec2::ZERO, Vec2::X, Vec2::ONE, Vec2::Y],
+            indices: vec![
+                1, 2, 3, // Tri 1
+                3, 0, 1, // Tri 2
+            ],
+        }
+    }
+
+    /// Computes mesh data for an hexagonal plane facing `Vec3::Y`
+    ///
+    /// Prefer using [`PlaneMeshBuilder`] for additional customization
+    #[must_use]
+    pub fn hexagonal_plane(layout: &HexLayout, hex: Hex) -> Self {
+        let center = layout.hex_to_world_pos(hex);
+        let center = Vec3::new(center.x, 0., center.y);
+        let corners = layout.hex_corners(hex);
+        let corners_arr = corners.map(|p| Vec3::new(p.x, 0., p.y));
+        Self {
+            vertices: vec![
+                center,
+                corners_arr[0],
+                corners_arr[1],
+                corners_arr[2],
+                corners_arr[3],
+                corners_arr[4],
+                corners_arr[5],
+            ],
+            uvs: vec![
+                UV_DELTA,
+                corners[0] + UV_DELTA,
+                corners[1] + UV_DELTA,
+                corners[2] + UV_DELTA,
+                corners[3] + UV_DELTA,
+                corners[4] + UV_DELTA,
+                corners[5] + UV_DELTA,
+            ],
+            normals: [Vec3::Y; 7].to_vec(),
+            indices: vec![
+                1, 0, 2, // 1
+                2, 0, 3, // 2
+                3, 0, 4, // 3
+                4, 0, 5, // 4
+                5, 0, 6, // 5
+                6, 0, 1, // 6
+            ],
+        }
+    }
+
+    /// Computes mesh data for an hexagonal column facing `Vec3::Y` without the bottom face
+    #[must_use]
+    #[deprecated(since = "0.6.0", note = "Use ColumnMeshBuilder instead")]
+    pub fn partial_hexagonal_column(layout: &HexLayout, hex: Hex, column_height: f32) -> Self {
+        ColumnMeshBuilder::new(layout, column_height)
+            .at(hex)
+            .without_bottom_face()
+            .build()
+    }
+
+    /// Computes mesh data for an hexagonal column facing `Vec3::Y`
+    #[must_use]
+    #[deprecated(since = "0.6.0", note = "Use ColumnMeshBuilder instead")]
+    pub fn hexagonal_column(layout: &HexLayout, hex: Hex, column_height: f32) -> Self {
+        ColumnMeshBuilder::new(layout, column_height)
+            .at(hex)
+            .build()
+    }
+
+    /// Computes cheap mesh data for an hexagonal column facing `Vec3::Y` without the bottom face.
+    ///
+    /// This mesh has only 13 vertices, as no vertex is duplicated. As a consequence the normals will behave strangely.
+    /// The UV mapping will be incorrect
+    ///
+    /// Use this mesh if you don't care about lighting and texturing.
+    #[must_use]
```

Comment:
This isn't deprecated: should we have a hex builder option for this instead?

Reply:
Agreed, I think it should be removed.

Reply:
Oh right, that could be useful for collision shapes as well